src/default_panels/StyleTracesPanel.js

```diff
@@ -564,6 +599,29 @@ const StyleTracesPanel = (props, {localize: _}) => (
         options={[{label: _('Show'), value: true}, {label: _('Hide'), value: false}]}
       />
     </PlotlySection>
+    <PlotlySection name={_('Totals Styles')}>
```

Comment:
can you rename this to "Total Marker Styles" and the other two from "X Trace Styles" to "X Marker Styles" plz?